components/login/login_controller/login_controller.jsx

```diff
@@ -639,7 +639,7 @@ class LoginController extends React.Component {
                     <Link to={'/reset_password'}>
                         <FormattedMessage
                             id='login.forgot'
-                            defaultMessage='I forgot my password'
+                            defaultMessage='I forgot my password.'
```

Comment:
Question: This sentence ends with a full-stop and I've noticed that none of the others do. Is this based on a different UI/context?

Reply:
The reason for this one ending with a period is, I wanted to make it similar to the corresponding text in the en.json file, where I used a period because this text is displayed as link text on the login page and is a complete sentence that looks odd without the punctuation. 
![image](https://user-images.githubusercontent.com/12574108/73641415-fb651a80-4695-11ea-8025-0e264cda6f6e.png)

(I added the ending punctuation to "Create one now" as well.)